app/app.js

```diff
@@ -1,60 +1,70 @@
-var app = angular.module("myCalculator", []);
+var app = angular.module("myCalculator", ['ngRoute']);
+app.config(['$routeProvider', function($routeProvider){
+	$routeProvider.when('/', {
+		templateUrl : 'templates/home.html',
+		controller : 'CalculatorCtrl'
+	}).when('/newMeal', {
+		templateUrl : 'templates/new-meal.html',
+		controller: 'CalculatorCtrl'
+	}).when('/earnings', {
+		templateUrl: 'templates/my-earnings.html',
+		controller: 'CalculatorCtrl'
+	});
```

Comment:
You should have a `.otherwise('/'` route, so there's a fallback if someone tries to go to a different route.